client/src/featureform/format.py

```diff
@@ -4,23 +4,16 @@
 divider = "-----------------------------------------------"
 
 def format_rows(format_obj, format_obj_2=None, format_obj_3=None, format_obj_4=None):
-    if not format_obj_2 and not format_obj_3:
+    if format_obj_2 is None:
```

Comment:
Here is another approach to do this and would love to hear your thoughts:

```
spacing_formats : {1: "{:<30} {:<25}", 2: "{:<30} {:<30} {:<30}", 3: "{:<30} {:<30} {:<30} {:<30}"}
divider = "-----------------------------------------------"

def format_rows(format_obj, *args):
    if len(args) == 0:
        for s in format_obj:
            format_rows(*s)
    else:
        format_args = (format_obj, ) + args
        format_string = spacing_formats.get(len(args))

        if format_string:
            print(format_string.format(*format_args))
        else:
            print(f"currently, format is not supported for {len(args)} fields")
```

What do you think? Is it cleaner or worse off?

*Don't need to implement it. It was just something I was thinking about and wanted to get your thoughts.*

Reply:
I can implement this in the next PR! Will add to my list :) Thank u!